src/main/java/wanya/task/Task.java

```diff
@@ -0,0 +1,94 @@
+package wanya.task;
+
+/**
+ * Represents a task to be completed.
+ */
+public abstract class Task {
+    private boolean hasCompleted = false;
+    private String taskName;
+
+    /**
+     * Creates a task when given a task name.
+     *
+     * @param taskName name of the task.
+     */
+    public Task(String taskName) {
+        this.taskName = taskName;
+    }
+
+    /**
+     * Creates a task when given a task name and completeness.
+     *
+     * @param taskName name of the task.
```

Comment:
I believe that you should capitalise the first letter of the word "name" in the @param line to adhere to Javadoc coding standards. I believe that there were other parts of the code in which you were unable to do so as well and I think it would be good to fix it just to be safe!